release/README.md

```diff
@@ -38,13 +42,42 @@ export REPO_PREFIX=gcr.io/bank-of-anthos
 ./make-release.sh
 ```
 
+This script does the following:
+1. Replaces the existing `kubernetes-manifests` with the contents of `dev-kubernetes-manifests`.
+2. Updates the image tag for all the Deployments in the new `kubernetes-manifests`, with the new release tag.
+3. Uses `git tag` to create a new local release.
+4. Creates a new release branch.
+5. Uses `skaffold` to build and push new stable release images to `gcr.io/bank-of-anthos`.
+6. Pushes the Git tags and the release branch.
+
+## Creating a PR
+
+Now that the release branch has been created, you can find it in the [list of branches](https://github.com/GoogleCloudPlatform/bank-of-anthos/branches) and create a pull request targeting `master` (the default branch.)
+
+This process is going to trigger multiple CI checks as well as stage the release onto a temporary cluster. Once the PR has been approved and all checks are successfully passing, you can now merge the branch.
+
+## Add notes to release
+
+Once the PR has been fully merged, you are ready to create a new release for the newly created [tag](https://github.com/GoogleCloudPlatform/bank-of-anthos/tags).
+
+The release notes should contain a brief description of the changes since the previous release (like bug fixed and new features.) For inspiration, you can look at the list of [releases](https://github.com/GoogleCloudPlatform/bank-of-anthos/releases).
```

Comment:
Tiny nit pick. Add the period after the closing brackets.
```
(like bug fixed and new features.)
```

Reply:
Aha! TIL. Somehow I always thought punctuation went _inside_ the brackets at all time.
Seems like the answer is "it depends": https://www.thepunctuationguide.com/period.html#placement-with-parentheses
I've fixed this.